ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -491,6 +491,12 @@ def __array__(self, dtype=None, /):
         if not dtype:
             return self
         return np_frontend.array(self, dtype=dtype)
+    
+    def __array_wrap__(self, array, context=None, /):
+        if context is None:
+            return self.view()
+        else:
+            return np_frontend.array(array)
```

Comment:
Here is when we have some context as it can be a dictionary carrying wrapping information we can use the leave it for the extension of other subclasses that will be overriding this method depending on the functionality needed `ndarray` class